docs/examples/getDataMoverResources.go

```diff
@@ -0,0 +1,319 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	appsV1 "k8s.io/api/apps/v1"
+	coreV1 "k8s.io/api/core/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/client-go/util/homedir"
+)
+
+var (
+	backup    bool
+	restore   bool
+	details   bool
+	namespace string
+
+	err error
+
+	ClusterConfig    *rest.Config
+	KubernetesClient *kubernetes.Clientset
+	DynamicClient    *dynamic.DynamicClient
+
+	VolumeSnapshotResource = schema.GroupVersionResource{
+		Group:    "snapshot.storage.k8s.io",
+		Version:  "v1",
+		Resource: "volumesnapshots",
+	}
+	VolumeSnapshotContentResource = schema.GroupVersionResource{
+		Group:    "snapshot.storage.k8s.io",
+		Version:  "v1",
+		Resource: "volumesnapshotcontents",
+	}
+	VolumeSnapshotBackupResource = schema.GroupVersionResource{
+		Group:    "datamover.oadp.openshift.io",
+		Version:  "v1alpha1",
+		Resource: "volumesnapshotbackups",
+	}
+	VolumeSnapshotRestoreResource = schema.GroupVersionResource{
+		Group:    "datamover.oadp.openshift.io",
+		Version:  "v1alpha1",
+		Resource: "volumesnapshotrestores",
+	}
+	ReplicationSourceResource = schema.GroupVersionResource{
+		Group:    "volsync.backube",
+		Version:  "v1alpha1",
+		Resource: "replicationsources",
+	}
+	ReplicationDestinationResource = schema.GroupVersionResource{
+		Group:    "volsync.backube",
+		Version:  "v1alpha1",
+		Resource: "replicationdestinations",
+	}
+)
+
+// TODO Does this work with built in DataMover?
```

Comment:
will test in my upcoming tasks

Reply:
tested

it works (it sees volumeSnapshotContents) but could be better

should I 
- merge like this (focused in OLD DataMover)
- add NEW DataMover stuff (DataUpload and DataDownload) and add new flag to select which to use
- remove OLD DataMover stuff (VSM and Volsync), and add NEW DataMover stuff
?